### PR TITLE
Mention about alternative keyserver on the index page

### PR DESCRIPTION
**Pull request by pkuczynski**
Fixes rvm/rvm#3438

### CODE REVIEW

File: content/index.haml [hunk 1]
```diff
@@ -17,24 +17,26 @@ title: RVM Ruby Version Manager - Documentation
 
 %ul.square
   %li
-    Install RVM:
+    Install GPG keys:
     = sh_cmd "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
```
Comment (mpapis) [inline content/index.haml]: maybe switch to `gpg2` without `--keyserver` and add a note to use `gpg` with it?

File: content/index.haml [hunk 2]
```diff
@@ -17,24 +17,26 @@ title: RVM Ruby Version Manager - Documentation
...
+
+  %li
+    Install RVM:
     = sh_cmd "\\curl -sSL https://get.rvm.io | bash -s stable"
-    For all in one installation append
-    %code --rails
-    or
-    %code --ruby
-    or
-    %code --ruby=1.9.3
```
Comment (mpapis) [inline content/index.haml]: I would leave it, having the `--rails` option is so important, nobody will dig for it ;) it's just one line

File: content/index.haml (hunk 1 above)
Comment (pkuczynski) [inline content/index.haml]: gpg2 does not need a keyserver anymore?

Comment (mpapis) [inline content/index.haml]: I'm not sure they ever needed it...

**Pull request merged**